components/checkbox/flag.js

```diff
@@ -0,0 +1,18 @@
+import h from 'virtual-dom/h';
+
+import Rx from 'rx';
+
+import {labelledCheckbox} from './labelled';
+
+
+const $combine = Rx.Observable.combineLatest;
+
+export function flagCheckbox(label, initialState = false) {
+    const cb = labelledCheckbox(label, initialState);
+    return {
+        model: {
+            value$: cb.model.checked$.map(checked => checked ? true : undefined)
+        },
+        tree$: $combine(cb.tree$, (...views) => h('span', views))
```

Comment:
Sure this doesn't need to be a `$combine`... perhaps `cb.tree$.map(view => h(span, [view]))`?